README.md

```diff
@@ -4,6 +4,14 @@ This repository contains Python notebooks and R Markdown guides for achieving sp
 
 Start learning with the [API Training](https://github.com/datarobot-community/tutorials-for-data-scientists/tree/master/DRU/API_Training) module.
 
+Additional code examples can be accessed from the following locations:
+
+- [Tutorials for data scientists repo](https://github.com/datarobot-community/tutorials-for-data-scientists/)
```

Comment:
this is also listed under "Important Links" - guessing you should remove that other link